src/main/java/net/sacredlabyrinth/phaed/simpleclans/managers/RequestManager.java

```diff
@@ -224,9 +224,8 @@ public void processInvite(Request req, VoteResult vote) {
 
         if (vote.equals(VoteResult.ACCEPT)) {
             ClanPlayer cp = plugin.getClanManager().getCreateClanPlayer(invited.getUniqueId());
-            int maxMembers = !clan.isVerified() ? plugin.getSettingsManager().getInt(CLAN_UNVERIFIED_MAX_MEMBERS) : plugin.getSettingsManager().getInt(CLAN_MAX_MEMBERS);
 
-            if (maxMembers > 0 && maxMembers > clan.getSize()) {
+            if (plugin.getSettingsManager().getInt(CLAN_MAX_MEMBERS) > 0 && clan.getMaxSize() > clan.getSize()) {
```

Comment:
why not use getMaxSize on both?

Reply:
I suppose that config value should have more effect than a dynamic flag